tests/test_sklearn_vectorizers.py

```diff
@@ -159,3 +161,74 @@ def test_explain_regression_hashing_vectorizer(newsgroups_train_binary):
         for (name, coef), (count_name, count_coef) in zip(values, count_values):
             assert name == count_name
             assert abs(coef - count_coef) < 0.05
+
+
+@pytest.mark.parametrize(['vec_cls'], [
+    [CountVectorizer],
+   #[HashingVectorizer],
+])
+def test_explain_feature_union(vec_cls):
+    data = [
+        {'url': 'http://a.com/blog',
+         'text': 'security research'},
+        {'url': 'http://a.com',
+         'text': 'security research'},
+        {'url': 'http://b.com/blog',
+         'text': 'health study'},
+        {'url': 'http://b.com',
+         'text': 'health research'},
+        {'url': 'http://c.com/blog',
+         'text': 'security'},
+        ]
+    ys = [1, 0, 0, 0, 1]
+    vec = FeatureUnion([
+        ('url', vec_cls(preprocessor=lambda x: x['url'],
+                        analyzer='char',
+                        ngram_range=(3, 3))),
+        ('text', vec_cls(preprocessor=lambda x: x['text'])),
+    ])
```

Comment:
I think showing it in a same document with the same scale (at least optionally) makes more sense, because it is easier to understand which parts of document are important overall. 

Yeah, this is what I had in mind for LIME - I was thinking about a default classifier to use, and 
a logistic regression trained on both char ngram and word unigram features looked like a baseline which should cover a lot of cases. I don't know how what it takes to implement this though :)